<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="Dell-Home-laptop" failures="6" tests="6" name="Login" time="97.73" errors="0" timestamp="2024-12-12T10:38:08 AEST">
  <testcase classname="testCases.TC007_ContactFormValidation" name="verify_ContactPage_Fields" time="10.227">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC007_ContactFormValidation.verify_ContactPage_Fields(TC007_ContactFormValidation.java:34)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verify_ContactPage_Fields -->
  <testcase classname="testCases.TC006_CoursesSubscriptionRadioButtons" name="Verify_SubscriptionPage_RadioButtons" time="10.196">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC006_CoursesSubscriptionRadioButtons.Verify_SubscriptionPage_RadioButtons(TC006_CoursesSubscriptionRadioButtons.java:38)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- Verify_SubscriptionPage_RadioButtons -->
  <testcase classname="testCases.TC001_LogInSuccess" name="Veiry_Login_Success" time="10.033">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC001_LogInSuccess.Veiry_Login_Success(TC001_LogInSuccess.java:41)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- Veiry_Login_Success -->
  <testcase classname="testCases.TC002_LogInFail" name="Verify_Login_Fail" time="10.063">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC002_LogInFail.Verify_Login_Fail(TC002_LogInFail.java:46)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- Verify_Login_Fail -->
  <testcase classname="testCases.TC005_PrivacyPolicyLink" name="Verify_PrivacyPolicyLink" time="10.374">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC005_PrivacyPolicyLink.Verify_PrivacyPolicyLink(TC005_PrivacyPolicyLink.java:33)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- Verify_PrivacyPolicyLink -->
  <testcase classname="testCases.TC004_TestException_ElementNotInteractableException" name="Verify_ElementNotInteractableException" time="10.176">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC004_TestException_ElementNotInteractableException.Verify_ElementNotInteractableException(TC004_TestException_ElementNotInteractableException.java:42)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- Verify_ElementNotInteractableException -->
  <testcase classname="testCases.TC003_LoginDataDriven" name="Verify_LoginByDataDriven" time="0.0">
    <ignored/>
  </testcase> <!-- Verify_LoginByDataDriven -->
</testsuite> <!-- Login -->
